python_modules/libraries/dagster-components/dagster_components/components/shim_components/sensor.py

```diff
@@ -3,12 +3,13 @@
 
 
 class SensorScaffolder(ShimScaffolder):
-    def get_text(self) -> str:
-        return """# import dagster as dg
+    def get_text(self, filename: str) -> str:
+        return f"""# import dagster as dg
 # 
```

Comment:
I assume this was litigated elsewhere, but curious why we comment these out by default. In the asset case specifically it'd be kind of nice for the stubbed asset to just show up in the UI

Reply:
I get the schedule/sensor cases being trickier tho cause they auto-run

Reply:
I think now that they have actual names it's more ok to have them uncommented, but my main reservation is actually that I really hate having code that is not formatted to my particular ruff's specifications automatically plopped in my codebase. Personal issue tho

Reply:
Agree with ben that we should default it to be being there and have a `--commented-out` function. Demoing is a super important use case here. (I also think we should strongly encourage custom scaffodling for real codebases at scale).